docs/sql-ref-syntax-qry-explain.md

```diff
@@ -19,4 +19,126 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `EXPLAIN` statement provides the execution plan for the statement. 
```

Comment:
How about `The EXPLAIN statement is used to provide logical/physical plans for an input statement. `?